AdyenDropIn/Models/PaymentMethodsConfiguration.swift

```diff
@@ -14,7 +14,7 @@ import PassKit
 public extension DropInComponent {
     
     /// Contains the configuration for the drop in component and the embedded payment method components.
-    final class PaymentMethodsConfiguration {
+    final class PaymentMethodsConfiguration: APIContextAware {
```

Comment:
DropInComponent is a component and already have `apiContext` "by definition"
How about move `apiContext` to DropIn's init, and not make  `PaymentMethodsConfiguration`  a  APIContextAware?

Reply:
that's a good point. but I think it's better to leave all configuration in for `DropInComponent` in `PaymentMethodsConfiguration`.
maybe we can do `public var apiContext: APIContext { configuration.apiContext }` in `DropInComponent` to show that it uses the configuration.

Reply:
👍